.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: bpfci
+
+on:
+  pull_request:
+
+jobs:
+  VM_Test:
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_NAME: 'libbpf'
+      AUTHOR_EMAIL: "$(git log -1 --pretty=\"%aE\")"
+      KERNEL: LATEST
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: install
+        run: for f in aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils; do sudo apt-get install $f ; done
+      - name: Kernel LATEST + selftests
+        run: export REPO_ROOT=$GITHUB_WORKSPACE ; export CI_ROOT=$REPO_ROOT/travis-ci; export VMTEST_ROOT=$CI_ROOT/vmtest ; env; ${CI_ROOT}/vmtest/run_vmtest.sh || exit 1
```

Comment:
just a random nit: I think REPO_ROOT is the only one that we should be specifying here, CI_ROOT and VMTEST_ROOT should be derivable inside run_vmtest.sh?

btw, `env;` is for debugging purposes?

Reply:
yeah, env was for debug
We arent using CI_ROOT var inside scripts at all, it's only for this oneliner in yaml, but VMTEST_ROOT is not something we redefining in our scripts, i think we have to keep it.